protobuf/v1/testing.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package mayastor.v1;
+
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/empty.proto";
+import "common.proto";
+
+service TestingRpc {
+  // Fault injection.
+  rpc AddInjection (AddInjectionRequest) returns (google.protobuf.Empty) {}
+  rpc RemoveInjection (RemoveInjectionRequest) returns (google.protobuf.Empty) {}
+  rpc ListInjections (ListInjectionsRequest) returns (ListInjectionsReply) {}
+}
+
+message Injection {
+  string uri = 1;
+  string device_name = 2;
```

Comment:
Do we need both here?

Reply:
URI defines all the parameters of an injection: target device, offset, time, fault type, etc.
Device name is just device part of it. It is given here for convenience, e.g. for CLI.
We'll probably want to add all the param here, once injections will be more or less stable.

Reply:
I see, thanks for clarifying.